test/Data/Graph/WrapperSpec.hs

```diff
@@ -21,3 +21,29 @@ spec = do
       it "returns the empty list" $ do
         reachableVertices (fromList [] :: Graph Integer ()) 0
           `shouldBe` []
+
+  describe "Eq Graph" $ do
+    context "when the graphs are identical" $ do
+      it "returns True" $ do
+        let g1 = fromList [('a','a',['a','b','c']), ('b','b',['b','c']), ('c','c',['a'])]
+        let g2 = fromList [('a','a',['a','b','c']), ('b','b',['b','c']), ('c','c',['a'])]
+        (g1 == g2)
+          `shouldBe` True
```

Comment:
I would put the `shouldBe` expression in one line. Also, I think, you don't need the parenthesis around `g1 == g2`, although I'm fine with keeping them, if you prefer.

Reply:
fixed.